{"name": "test_login_case", "status": "failed", "statusDetails": {"message": "AssertionError: assert '登录成功！' == '登录成功'\n  - 登录成功\n  + 登录成功！\n  ?     +", "trace": "self = <PrimarySelenium.case.login.test_login_case.TestLoginCase object at 0x03B7DFE8>, init_login = <page.login_page.LoginPage object at 0x03B9B388>\n\n    @pytest.mark.usefixtures(\"init_driver\")\n    def test_login_case(self, init_login):\n        # 登录\n        init_login.login(\"13710646151\", \"123456\")\n>       assert init_login.check_login_success() == \"登录成功\"\nE       AssertionError: assert '登录成功！' == '登录成功'\nE         - 登录成功\nE         + 登录成功！\nE         ?     +\n\ncase\\login\\test_login_case.py:12: AssertionError"}, "attachments": [{"name": "log", "source": "08d17cf6-69b5-476e-868f-6b5457cbbf99-attachment.txt", "type": "text/plain"}], "start": 1597290944274, "stop": 1597290946609, "uuid": "8ebccea2-85fb-4df0-b3d9-9ee1c79bf606", "historyId": "0e185d3b795b3f6938e308f78ff1dd1a", "testCaseId": "60e7ad12e8b27bb7f15e22579b8401b6", "fullName": "case.login.test_login_case.TestLoginCase#test_login_case", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "case.login"}, {"name": "suite", "value": "test_login_case"}, {"name": "subSuite", "value": "TestLoginCase"}, {"name": "host", "value": "FYYX-2019VXOSDQ"}, {"name": "thread", "value": "11280-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.login.test_login_case"}]}